spec/index.html

```diff
@@ -1350,106 +1445,77 @@ <h3>The <code>rdf:XMLLiteral</code> Datatype</h3>
-
-  <p>In this way, the RDF-bearing representation acts as an intermediary
-    between the web-accessible primary resource, and some set of possibly
-    non-web or abstract entities that the <a>RDF graph</a> may describe.</p>
-
-  <p>In cases where other specifications constrain the semantics of
-    <a>fragment identifiers</a> in RDF-bearing representations, the encoded
-    <a>RDF graph</a> should use fragment identifiers in a way that is consistent
-    with these constraints. For example, in an HTML+RDFa document [[HTML-RDFA]],
-    the fragment <code>chapter1</code> may identify a document section
-    via the semantics of HTML's <code>@name</code> or <code>@id</code>
-    attributes. The <a>IRI</a> <code>&lt;#chapter1&gt;</code> should
-    then be taken to <a>denote</a> that same section in any RDFa-encoded
-    <a>triples</a> within the same document.
-    Similarly, fragment identifiers should be used consistently in resources
-    with multiple representations that are made available via
-    <a data-cite="webarch/#frag-coneg">content negotiation</a>
-    [[WEBARCH]]. For example, if the fragment <code>chapter1</code> identifies a
-    document section in an HTML representation of the primary resource, then the
-    <a>IRI</a> <code>&lt;#chapter1&gt;</code> should be taken to
-    <a>denote</a> that same section in all RDF-bearing representations of the
-    same primary resource.</p>
-</section>
-
-<section id="section-generalized-rdf" class="informative">
-  <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
+    <dl>
+      <dt id="JSON-uri">The IRI denoting this <a>datatype</a></dt>
+      <dd>is <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#JSON</code>.</dd>
 
-  <p>It is sometimes convenient to loosen the requirements
-    on <a>RDF triple</a>s.  For example, the completeness
-    of the RDFS entailment rules is easier to show with a
-    generalization of RDF triples.</p>
+      <dt id="JSON-lexical-space">The <a>lexical space</a></dt>
+      <dd>is the set of all <a>strings</a> that conform to the
+        <a data-cite="RFC4627#section-2">JSON Grammar</a> as described in
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
+        [[RFC8259]].</dd>
 
-  <p>A <dfn class="export">generalized RDF triple</dfn> is a triple having a subject,
-    a predicate, and object, where each can be an <a>IRI</a>, a
-    <a>blank node</a> or a
-    <a>literal</a>. A
-    <dfn class="export">generalized RDF graph</dfn>
-    is a set of generalized RDF triples. A
-    <dfn class="export">generalized RDF dataset</dfn>
-    comprises a distinguished generalized RDF graph, and zero
-    or more pairs each associating an IRI, a blank node or a literal
-    to a generalized RDF graph.</p>
+      <dt id="JSON-value-space">The <a>value space</a></dt>
+      <dd>
+        is the space defined by
+        the <a data-cite="JSON-LD11#dfn-internal-representation">JSON-LD internal representation</a> [[JSON-LD11]],
```

Comment:
This probably works on its own, without reference to the JSON-LD internal representation.

Reply:
It's not clear to me what "this" is that "probably works on its own". I suggest a change request, to make it clear what you mean.

Reply:
My suggestion was that we could probably drop the reference to JSON-LD internal representation, as we pretty much describe the INFRA/ECMAScript types we use directly. There was a concern raised at TPAC about layering specs.